interfaces/ifacetest/testiface.go

```diff
@@ -104,6 +105,16 @@ type TestInterface struct {
 	SystemdPermanentSlotCallback func(spec *systemd.Specification, slot *snap.SlotInfo) error
 }
 
+// TestHotplugInterface is a interface for various kind of tests needing hotplug-aware interface.
```

Comment:
I think you can move this just above the first method of this struct.

Reply:
Added a new comment above first method, left this one as this is consistent with TestInterface defined above.